.travis.yml

```diff
@@ -15,4 +23,6 @@ branches:
   - "/^greenkeeper/.*$/"
 env:
   global:
-    secure: HN9UwIwZ0NIOgGaRHssKraQy5L4q0Jagtq4nsw75vUHmrc2tNYCpqREtIr7DlapurtVpXsOd5G8FiVMN6YWQA1WVuvoTD7AqJOXYF72gtSnIJ+mER6a4fFQe+2cPMu8l6/jYAFDzQ/ZJqd7QM8DZu6siYAmiJqmq1OmUSP3aohn/bCwyKcvD8MybNQvYvB9j1YzcypbE1LFUeV0oQX8rt9F+5PgY0vB/JE+X806sepgdv8LiVAfy2D/4IhHE55QtC1dfKsCl7Sn1nVriD16Z+OwsabQnp61wptOxaOsHM41eaXd1YlRyqYM1czNoFqgPMyo8uycSO/c7zl5cxWadZmvl21aZXHkb2HTH/XKP6BaZAa7DHwt1S2M8tP9YLDwhO24kSbxRA53Eiqnx+i3kgqsE+g2/XN7dMZpJ3MKgxyXB/dMC1G9qORGs1HdBgDIs4PXP4RMssXpntYJ+QW5JN/cByyA/K90V4K+QV9I4kVVBIXYvTyoyaTj/DwCXwC0zzU+F0BDcTHLtndFvJw6LND/WcYYKTJvwP0glPD/7okPtjYXzWMYPC0jkazLc85g31YvKDvfIel+OJ0j3Z/d2mbekIrL4j1+OSGv2rqSVaLT2H+QUO2WJaYvP8J0ITBVRSp5TopJL9Td9OOhIG/t6FxgKRDnKmoK3647NzcY4TFI=
+  - BUILD_LEADER_ID=1
+  - secure: HN9UwIwZ0NIOgGaRHssKraQy5L4q0Jagtq4nsw75vUHmrc2tNYCpqREtIr7DlapurtVpXsOd5G8FiVMN6YWQA1WVuvoTD7AqJOXYF72gtSnIJ+mER6a4fFQe+2cPMu8l6/jYAFDzQ/ZJqd7QM8DZu6siYAmiJqmq1OmUSP3aohn/bCwyKcvD8MybNQvYvB9j1YzcypbE1LFUeV0oQX8rt9F+5PgY0vB/JE+X806sepgdv8LiVAfy2D/4IhHE55QtC1dfKsCl7Sn1nVriD16Z+OwsabQnp61wptOxaOsHM41eaXd1YlRyqYM1czNoFqgPMyo8uycSO/c7zl5cxWadZmvl21aZXHkb2HTH/XKP6BaZAa7DHwt1S2M8tP9YLDwhO24kSbxRA53Eiqnx+i3kgqsE+g2/XN7dMZpJ3MKgxyXB/dMC1G9qORGs1HdBgDIs4PXP4RMssXpntYJ+QW5JN/cByyA/K90V4K+QV9I4kVVBIXYvTyoyaTj/DwCXwC0zzU+F0BDcTHLtndFvJw6LND/WcYYKTJvwP0glPD/7okPtjYXzWMYPC0jkazLc85g31YvKDvfIel+OJ0j3Z/d2mbekIrL4j1+OSGv2rqSVaLT2H+QUO2WJaYvP8J0ITBVRSp5TopJL9Td9OOhIG/t6FxgKRDnKmoK3647NzcY4TFI=
+  - secure: MBK8f97y4VozvQtJAPkFdoeyKyEfoX9mln/HE4AaAm2XB+xmCVQxFayTcfJoqhL4ZYh4xKrRIndxnnVHexc9aLuJKooC9azBU7jyQXIEKiGxlvo6Y2BOmsWLDZl41KIwfjWRoyCV1Of6Bf2RFyZg1Yut2RGtjYHTD6P4jRKwBNEpE9MyBJLQYmJw0KEwYQiY4Jt3fZtouSgyZS2jBeiBQEyS9mShbISdtjqUN021H/zpdfqvD0UUAehu/PhztiD2x0d23GkcLASMeBUlnrf9Vy5JmRFmrFgw9i4BDg6i1pDooV6xXShV5VoMmtPmSAuCkDvrGI8ETFqUB9799mHmyd1qcjHNWN+xojKfDVE+5G0AnyHWzHwI7PixDhWdNREKhuk8un5ACG+r3EZKyLkfN43RLoid+OHIaZk8Gfo+12Z4b9Z2v5ixjOvD/K8CIC3EKxee4iYRE4QTfeHf5NMoC5lfzOFVfcb84w5I12RK8dHiLRtQa5x6R665w5wGtQQWpRhaXMb4qOcA4jD3WE/g+ugnYJU18uDS6Awlg3MKKhRo+l+T7neusAZTLlqlAp0Y4rvZ9BlJRjIyeIxgC9yTI4Gz+daiskVtNuNh9yAv38MA3q1HP3rfRU1T21SmLBNMRe9uaDARk5jPDjUDQEYfMoGVt78xn+atwz1YdKTG1oA=
```

Comment:
I believe these can go into environment variables in Travis, no?

Reply:
It's less work to run `travis encrypt` locally :P.

What's the advantage of having them in repo settings?